farm/eval.py

```diff
@@ -7,14 +7,23 @@
 from sklearn.metrics import classification_report
 from torch.utils.data import DataLoader
 
-from farm.metrics import compute_metrics
+from farm.metrics import compute_metrics, _correct_bio_encodings
 from farm.utils import to_numpy
 from farm.utils import MLFlowLogger as MlLogger
 from farm.modeling.adaptive_model import AdaptiveModel
 
 logger = logging.getLogger(__name__)
 
 
+def _token_classification_report(labels, preds, digits):
+    preds = [
+        [token if token not in {"X", "[PAD]"} else "O" for token in sent]
```

Comment:
Are you sure this is still needed? Even when removing these tokens from our label list?